src/Control/Monad/Eff/Console/Unsafe.purs

```diff
@@ -0,0 +1,13 @@
+-- | Dodgy functions to output runtime representation of arbitrary values.
```

Comment:
No I think it's right, but I use the term in the PureScript book, and it's not quite the same thing. The way I've used it previously implies structure all the way to the leaves of the tree of values.